OpenFusion.vcxproj.filters

```diff
@@ -0,0 +1,204 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this from a fork? This shouldn't be pushed upstream. [4]

Reply:
fork